src/NodaTime.Web/Startup.cs

```diff
@@ -80,8 +85,10 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 ContentTypeProvider = new FileExtensionContentTypeProvider
                 {
                     Mappings = { [".nzd"] = "application/octet-stream" }
-                }
+                },
+                OnPrepareResponse = (context) => SetCacheControlHeaderForStaticContent(env, context.Context)
```

Comment:
Nit: no need for the brackets around `context`. Should have spotted that before.